server/build.gradle

```diff
@@ -48,6 +49,60 @@ configurations {
     }
 }
 
+class Rspec extends DefaultTask {
+
+    Rspec() {
+        description = 'Run ruby based spec tests'
+        group = 'Verification'
+    }
+
+    @Input
+    String test = ""
+
+    @Input
+    String spec = ""
+
+    @Option(option = 'spec', description = 'The name of the spec file to search. Example: "core_and_ram"')
+    void setSpec(final String specfile_name) {
+        this.spec = specfile_name
+    }
+
+    @Option(option = 'test', description = 'Set the name of the spec test to run')
+    void setTest(final String test_name) {
+        this.test = test_name
+    }
+
+    @TaskAction
+    runRspec() {
+        def rspec_args = [
+                "--format", "documentation",
```

Comment:
Can we add "--force-color"?  That gets the colors back in the output making it easy to tell when a test fails (since it prints red)